proto/viam/app/v1/app.proto

```diff
@@ -1280,6 +1308,7 @@ message UploadModuleFileResponse {
 message GetModuleRequest {
   // The id of the module (formatted as prefix:name where prefix is the module owner's orgid or namespace)
   string module_id = 1;
+  optional bool include_markdown_documentation = 2;
```

Comment:
[just curious] do we have any plans to eventually deprecate `GetModule`?

Reply:
No official plan but I do want to migrate the CLI to use the registry item APIs. Depends on effort. I was going to do it in the fancy config project. I'm going to pickup the CLI changes for this project next week and consider doing it then.

One of the blocking things was to add the `ModuleUpdateMetadata` message which is one reason why I included in this PR so that I have the option to do it.

Reply:
As soon as the CLI is off, the UI would need to migrate. And then probably give it another ~2-3mo for people to naturally upgrade to new versions of the CLI.